homeassistant/components/nanoleaf/const.py

```diff
@@ -7,8 +7,12 @@
 TOUCH_MODELS = {"NL29", "NL42", "NL52"}
 
 TOUCH_GESTURE_TRIGGER_MAP = {
+    0: "single_tap",
+    1: "double_tap",
     2: "swipe_up",
     3: "swipe_down",
     4: "swipe_left",
     5: "swipe_right",
 }
+
+NANOLEAF_PANEL_ID = "panel_id"
```

Comment:
It does not appear that this can be translated when displayed in the Trigger/Actions UI. A few other integrations have comments indicating the same thing; should I add one here?